Kiosk/App/CardHandler.swift

```diff
@@ -70,25 +72,25 @@ class CardHandler: NSObject, CFTReaderDelegate {
     // handle other delegate call backs with the status messages
 
     func readerIsAttached() {
-        _cardStatus.onNext("Reader is attatched");
+        _cardStatus.onNext("Reader is attatched")
     }
 
     func readerIsConnecting() {
-        _cardStatus.onNext("Reader is connecting");
+        _cardStatus.onNext("Reader is connecting")
     }
 
     func readerIsDisconnected() {
-        _cardStatus.onNext("Reader is disconnected");
+        _cardStatus.onNext("Reader is disconnected")
         logger.log("Card Reader Disconnected")
     }
 
     func readerSwipeDidCancel() {
-        _cardStatus.onNext("Reader did cancel");
+        _cardStatus.onNext("Reader did cancel")
         logger.log("Card Reader was Cancelled")
     }
 
     func readerGenericResponse(_ cardData: String!) {
-        _cardStatus.onNext("Reader received non-card data: \(cardData ?? "") ");
+        _cardStatus.onNext("Reader received non-card data: \(cardData ?? "") ")
```

Comment:
Is `semi: true` supported?

Reply:
Right now semis are confusing the AST parser and removing them is actually necessary to get a good AST.

In the long run, while there's a split in JS community on whether semis should be used (the ASI has its quirks) I'd prefer to embrace prettier's opinionatedness and not offer the `semi` option for Swift at all.

Reply:
That's something I want to encourage with plugins, just because Prettier for JS has an option for a thing that is applicable to another language, doesn't mean it should be added to that language. The options are largely about improving the adoptability of Prettier, so if `semi: true` won't be missed, don't add it!

Reply:
@azz 👍  I actually do support `commas: "all"` though (I might be biased here).

Reply:
Agree on no option, I've never seen Swift design guides that have semis, it's not like the JS world split. People just don't do it in Swift. 

This one was probably just that we'd been working in objc-c and got used to it ;)